### PR TITLE
triple equals

### PR DESCRIPTION
**Pull request by tolbon**
Maybe a too big commit but just add an other equals for strict type checking

I modify core/Updater/Migration/Db/DropColumn.php with delete unnecessary code cause placeColumnAfter never exist in this scope

### CODE REVIEW

**Comment by tolbon** [comment]
I modify == to === in 2 cases
`if(count($var) == fix_number)` 
or 
`if($va) == fix_string`

File: core/Updater/Migration/Db/DropColumn.php [hunk 1]
```diff
@@ -17,10 +17,6 @@ public function __construct($table, $columnName)
     {
         $sql = sprintf("ALTER TABLE `%s` DROP COLUMN `%s`", $table, $columnName);
 
-        if (!empty($placeColumnAfter)) {
-            $sql .= sprintf(' AFTER `%s`', $placeColumnAfter);
-        }
-
         parent::__construct($sql, static::ERROR_CODE_COLUMN_NOT_EXISTS);
```
Comment (Findus23) [inline core/Updater/Migration/Db/DropColumn.php:20]: This change doesn't seem to fit with the rest. Was it intentional?

Comment (sgiehl) [reply core/Updater/Migration/Db/DropColumn.php:20]: as he had written in the PR description:

> I modify core/Updater/Migration/Db/DropColumn.php with delete unnecessary code cause placeColumnAfter never exist in this scope

Comment (Findus23) [reply core/Updater/Migration/Db/DropColumn.php:20]: Sorry, didn't see that

**Comment by tsteur** [comment]
Thanks @tolbon just fixed some merge conflicts and looks good to merge once tests pass 👍

**Pull request merged**